01week/rockPaperScissors.js

```diff
@@ -10,14 +10,47 @@ const rl = readline.createInterface({
 
 function rockPaperScissors(hand1, hand2) {
 
-  // Write code here
+  //3. Compare User1 input to User2 input. (if else) (Combine hand1 and hand 2)
+  const combinedHands = hand1 + hand2;
```

Comment:
hand1 and hand2 should be forced to lowercase with no whitespace.  Currently you are writing to the console when an invalid input is passed.  I.e, rockPaperScissors('rock ', 'sCiSsOrs'), should return "Hand one wins!"